playbooks/collect_activity/collect_activity.yml

```diff
@@ -15,13 +15,13 @@
       copy:
         src: stats.py
         dest: /tmp/collect_activity.py
-      become_user: www-data
+      become: yes
```

Comment:
It looks like www-data doesn't have even read access for any of the requisite files anymore, and I figured that, since this is just a test, it was a better choice to just use a user that'll always have the necessary privileges than to spend time figuring out who has those privileges now, to be further updated as needed.